benchopt/utils/checkers.py

```diff
@@ -65,3 +67,24 @@ def solver_supports_dataset(solver, dataset):
         return False
 
     return True
+
+
+def check_random_state(seed):
```

Comment:
honest question, is it better to implement ours than to import it from sklearn.utils ?